linalg/solvers.cpp

```diff
@@ -1317,7 +1317,9 @@ void FGMRESSolver::Mult(const Vector &b, Vector &x) const
       if (v[i]) { delete v[i]; }
       if (z[i]) { delete z[i]; }
    }
-   converged = false;
+
+   final_norm = beta;
+   final_iter = j;
```

Comment:
Small tweak: if `converged` is `false`, then `j` will be `max_iter+1`, however, we actually want `final_iter` to be `max_iter`.
```suggestion
   final_iter = converged ? j : max_iter;
```

Reply:
Done + corrected the printout 😉 .